manifests/charts/istio-cni/values.yaml

```diff
@@ -19,7 +19,7 @@ _internal_defaults_do_not_set:
   # CNI-and-platform specific path defaults.
   # These may need to be set to platform-specific values, consult
   # overrides for your platform in `manifests/helm-profiles/platform-*.yaml`
-  cniBinDir: /opt/cni/bin
```

Comment:
Don't change this, change the `gke` default in `manifests/helm-profiles/platform-gke.yaml`:

```
cni:
  cniBinDir: /opt/cni/bin
```

Reply:
if we dont remove the global default the below condition (.Values.cniBinDir) will always be true. This will override the detected gke cni bin dir.

```
{{- if .Values.cniBinDir }}
{{ $detectedBinDir = .Values.cniBinDir }}
{{- end }}
```

Reply:
Would this cause regression?
I think previously `helm install istio-cni` or anyother way of simple install would do autodetection and now it wouldn't. I can make the change if this is not an issue.

Reply:
> I think previously helm install istio-cni or anyother way of simple install would do autodetection

1. This autodetection is by-definition only performed on `gke`
2. Installing on `gke` without using `--platform=gke` is not supported

So no, it shouldn't be a regression from supported behavior.

Reply:
Updated the files